joomla/joomla-cms
# [4.0] Merge core and com_media builds

Pull Request for Issue # .

### Summary of Changes

`com_media` has it's own `package.json` and build scripts. This PR merges them into core. They're now in a single location without any version mismatches.

Some of the dependencies needs to be bumped up


### Testing Instructions

1. Run `npm i` and ensure there are no errors
2. Look at the Media Manager in the backend. Ensure there are no console errors and the page displays properly.
